sirepo/job_supervisor.py

```diff
@@ -290,14 +290,14 @@ def _get_uids_and_files():
             if r:
                 yield u, r
 
-        def _purge_sim(db_file):
-            d = pkcollections.json_load_any(db_file)
+        def _purge_sim(compute_jid):
+            d = cls.__db_load(compute_jid)
```

Comment:
Why is this called cmopute_jid, when elsewhere it's just jid.

Reply:
https://github.com/radiasoft/sirepo/pull/2534/commits/9ce89be2fa5fb91607e6a20d55f8bf0cb9a69f45